.github/workflows/go-integration.yml

```diff
@@ -4,9 +4,9 @@ on:
   # The Push runners always seem to be slower.
   # Seems to be due to hitting cache limits and only one runner
   # being able to access the cache at a time.
-  # push:
-  #   branches-ignore:
-  #     - master
+  push:
```

Comment:
It probably would be worth figuring out why this was commented out?  

@jayofdoom or @richscott may have some context?

Reply:
I don't know about this, but `git blame` on this file shows that @Sharpz7 commented those out and added the commentary lines above on May 12, 2023 (prior to that, the directives were enabled, last written by @jayofdoom in August of last year). 

I don't know about "Push runners" - does GH demarcate runner pools (that they issue jobs to) by event type?

Reply:
Essentially my best bet is it comes down to something like there not being the cache space for both workflows at the same time. Even then, why do we have 2 workflows? Probably a question for Jay?

Reply:
Added it back as the e2e tests didn't seem to run.